README.md

```diff
@@ -1 +1,29 @@
-# team-culture
\ No newline at end of file
+# Code of Conduct
+
+## Diversity
+We accept others, no matter the gender, nationality, culture, religion.
+
+## Respect
+We all have different backgrounds, beliefs and points of view.
+Treat others as you wish to be treated.
+
+## Trust and Cooperation
+We are a team, don't be afraid to be judged. We support each other.
+
+## Autonomy
+You are the owner of your task, you choose how to do your tasks , when to ask for help, and when to pull the plug on a discussion that you asked for.
+
+## Responsability
+You are responsible for your actions. But as a team, we all are responsible for the product and for what we deploy.
+
+## Ownership
+Be the owner of your code and projects.  If something fails on it, you have to take care of it.
+
+## Disagree
+You don't have to agree with everything we say, it is ok to disagree, give your arguments and let's set up an agreement.
+
+## Commit
```

Comment:
## Disagree and Commit
It is not always necessary that you agree with everything and everyone in order to commit to the task. Feel free to discus it and make your points, but you are expected to be reliable on delivering high quality work, even if you don't agree with the outcome.